src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -170,11 +170,65 @@ public override void ConfigureServices(IServiceCollection services)
         }
     }
 
+    [Feature(OpenIdConstants.Features.Management)]
```

Comment:
@sebastienros bad merge?

Reply:
What I am asking, probably

Reply:
None of the proposed changes makes sense, so I'd revert the entire file changes.

Reply:
ok, reverted changes in startup.cs and added the recipe step handlers